client/src/components/PostCard.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import { Card, Space, Typography } from 'antd';
+import { Link } from 'react-router-dom';
+import Image from './Image';
+import '../styles/post-card.css'
+import Carousel from './Carousel';
+import PostButtons from './PostButtons';
+import PostComments from './PostComment';
+import CommentForm from './CommentForm';
+import PostDetails from './PostDetails';
+import formatTime from '../helpers/timeFormater';
+import { IPost } from '../interfaces';
+
+const { Paragraph } = Typography;
+interface Props {
+    post: IPost
+}
+const PostCard: React.FC<Props> = ({ post }) => {
+    const [showComments, setShowComments] = useState(false);
+    const [showLike, setShowLike] = useState(false);
+    const comments = [
```

Comment:
maybe you should move these to some JSON file and pass them as props to the component